cygnus-ngsi/src/main/java/com/telefonica/iot/cygnus/aggregation/NGSIGenericAggregator.java

```diff
@@ -0,0 +1,297 @@
+/**
+ * Copyright 2014-2017 Telefonica Investigación y Desarrollo, S.A.U
```

Comment:
```suggestion
 * Copyright 2014-2020 Telefonica Investigación y Desarrollo, S.A.U
```

:D

Reply:
Fixed in [0d50fd2](https://github.com/telefonicaid/fiware-cygnus/pull/1798/commits/0d50fd2a1dd53ec6084568e6e2ce90bb3f64daba)